packages/finoaconnect/package.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "@web3-onboard/finoaconnect",
+    "version": "2.0.0-alpha.1",
+    "description": "FinoaConnect enables DApp users to use Finoa's Institutional Custody services.",
+    "keywords": [
+      "Ethereum",
+      "Web3",
+      "EVM",
+      "dapp",
+      "Multichain",
+      "Wallet",
+      "Transaction",
+      "Provider",
+      "Hardware Wallet",
+      "Notifications",
+      "React",
+      "Svelte",
+      "Vue",
+      "Next",
+      "Nuxt",
+      "MetaMask",
+      "Coinbase",
+      "WalletConnect",
+      "Ledger",
+      "Trezor",
+      "Connect Wallet",
+      "Ethereum Hooks",
+      "Blocknative",
+      "Mempool",
+      "pending",
+      "confirmed",
+      "Injected Wallet",
+      "Crypto",
+      "Crypto Wallet",
+      "Tally Ho",
+      "FinoaConnect"
+    ],
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/blocknative/web3-onboard.git",
+      "directory": "packages/finoaconnect"
+    },
+    "homepage": "https://onboard.blocknative.com",
+    "bugs": "https://github.com/blocknative/web3-onboard/issues",
+    "module": "dist/index.js",
+    "browser": "dist/index.js",
+    "main": "dist/index.js",
+    "type": "module",
+    "typings": "dist/index.d.ts",
+    "files": [
+      "dist"
+    ],
+    "scripts": {
+      "build": "tsc",
+      "dev": "tsc -w",
+      "type-check": "tsc --noEmit"
+    },
+    "devDependencies": {
+      "typescript": "5.1.6"
```

Comment:
Can we update to the latest version of typescript here? We have a feature in the pipeline that will require this

Reply:
resolved in latest commit